specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/gallery.json

```diff
@@ -0,0 +1,1480 @@
+        }
+      },
+      "description": "Describes the gallery Image Definition purchase plan. This is used by marketplace images."
+    },
+    "GalleryImageVersion": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/GalleryImageVersionProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "description": "Specifies information about the gallery Image Version that you want to create or update."
+    },
+    "GalleryImageVersionProperties": {
+      "properties":{
+        "publishingProfile": {
+          "$ref": "#/definitions/GalleryImageVersionPublishingProfile"
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "title": "The current state of the gallery Image Version.",
+          "description": "The provisioning state, which only appears in the response.",
+          "enum": [
+            "Creating",
+            "Updating",
+            "Failed",
+            "Succeeded",
+            "Deleting",
+            "Migrating"
+          ]
+        },
+        "storageProfile": {
+          "readOnly": true,
+          "$ref": "#/definitions/GalleryImageVersionStorageProfile"
+        },
+        "replicationStatus": {
+          "readOnly": true,
+          "$ref": "#/definitions/ReplicationStatus"
+        }
+      },
+      "required": [
+        "publishingProfile"
+      ],
+      "description": "Describes the properties of a gallery Image Version."
+    },
+    "GalleryArtifactPublishingProfileBase": {
+      "properties": {
+        "targetRegions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TargetRegion"
+          },
+          "description": "The target regions where the Image Version is going to be replicated to. This property is updatable."
+        },
+        "source": {
+          "$ref": "#/definitions/GalleryArtifactSource"
+        }
+      },
+      "required": [
+        "source"
+      ],
+      "description": "Describes the basic gallery artifact publishing profile."
+    },
+    "TargetRegion": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the region."
+        },
+        "regionalReplicaCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The number of replicas of the Image Version to be created per region. This property is updatable."
+        },
+		"storageAccountType": {
+          "type": "string",
+          "description": "Specifies the storage account type to be used to store the image. This property is not updatable.",
+          "enum": [
+            "Standard_LRS",
+            "Standard_ZRS"
+          ],
+          "x-ms-enum": {
+            "name": "StorageAccountType",
+            "modelAsString": false
```

Comment:
Is there a strong reason for modelAsString: false?  If you want to be able to add values within an API version, I believe this must be `true`

Reply:
Ah I see, I've updated to true. Thanks.